pages/linux/dex.md

```diff
@@ -0,0 +1,32 @@
+# dex
+
+> DesktopEntry Execution is a program to generate and execute DesktopEntry files of the Application type.
+> More information: <https://github.com/jceb/dex>.
+
+- Perform an autostart/execute all programs in the autostart folders:
+
+`dex --autostart`
+
+- Perform an autostart/execute all programs in the specified folders:
+
+`dex --autostart --search-paths {{path/to/directory}}:{{path/to/directory}}`
+
+- Preview the programs would be executed in a regular autostart:
```

Comment:
With preview, do you mean print the names in the terminal?

Reply:
tbh; only documenting this, did not try the application.